### DISCUSSION TITLE
Flame GUI support

### DISCUSSION THREAD

**Discussion by cmedamine** [flame-engine/flame#1842] [General]
Will there be a GUI for flame like a web app in the future?

**Comment by spydon** [comment]
Do you mean like an editor, or what do you mean by a web app?

**Reply by cmedamine** [reply]
I mean like just like if you are familiar with buildbox, or game maker studio they use low/no code for binding things like characters...etc.
I mean you kind of have like software that generates dart code in the background by just drag and drop like stuff, and thank you for responding I'm just curious about this because it will be awesome if have some kind of this experience with flutter and flame in the future.

**Reply by spydon** [reply]
There is no real plan for it in the roadmap, but I know that @erickzanardo have spoken about evolving FireAtlas to contain some editor features in the long run.

**Reply by cmedamine** [reply]
that's great to hear thanks!
